just-install.json

```diff
@@ -157,6 +157,20 @@
       },
       "version": "2.2"
     },
+    "dropbox": {
+      "installer": {
+        "kind": "custom",
+        "options": {
+          "arguments": [
+            "{{.installer}}",
+            "/S"
+          ],
+          "extension": ".exe"
+        },
+        "x86": "https://www.dropbox.com/download?type=full&build={{.version}}&plat=win"
+      },
+      "version": "3.12.6"
```

Comment:
I would remove the `build=...` parameter from the URL and change the "version" field to "latest", this way users always get the latest version.